Makefile

```diff
@@ -34,12 +38,10 @@ generate:
 build-test:
 	@$(CONTAINER_ENGINE) build -t $(IMAGE_TEST) -f dockerfiles/Dockerfile.test .
 
-test-app: build-test
-#	Target to test app with tox on docker
+test-app: build-test ## Target to test app with tox on docker
 	@$(CONTAINER_ENGINE) run --rm $(IMAGE_TEST)
 
-test-container-image: build
-#	Target to test the final image
+test-container-image: build ## Target to test the final image
```

Comment:
Leftover from a previous PR? Please rebase. :)

Reply:
This is actually intended. This comment gets printed on the `help` target. The regex currently requires the comment to be on the same line as the target.